src/defaults.js

```diff
@@ -18,6 +18,12 @@ var propClassLayersMap = {
     NO_ROWS: [propClassEnum.COLUMNS, propClassEnum.CELLS]
 };
 
+var CellEditorModeEnum = {
```

Comment:
We added a Cell Editor Mode.
It is part of a cell config, by default this is disabled.